samza-core/src/main/java/org/apache/samza/config/StorageConfig.java

```diff
@@ -131,8 +140,8 @@ public String getStorageMsgSerde(String storeName) {
    *
    * @return the name of the system to use by default for all changelogs, if defined.
    */
-  public String getChangelogSystem() {
```

Comment:
Fix the following first line of `getChangelogSystem` javadoc:
`Gets the System to use for reading/writing checkpoints`